asab/storage/mongodb.py

```diff
@@ -189,6 +189,9 @@ async def execute(self, custom_data: typing.Optional[dict] = None):
 			if custom_data is not None:
 				webhook_data["custom"] = custom_data
 
+			if event_type is not None:
+				webhook_data["custom"]["event_type"] = event_type
```

Comment:
this will crash with KeyError if `custom_data is None`

besides, it might be a better idea to move the event_type outside `"custom"`, right into `webhook_data["event_type"]`